src/test/pages/AssignmentProgress.test.tsx

```diff
@@ -1,6 +1,6 @@
 import {screen} from "@testing-library/react";
 import {renderTestEnvironment, followHeaderNavLink} from "../utils";
-import {API_PATH, siteSpecific} from "../../app/services";
```

Comment:
Similarly, I guess that `siteSpecific` will be removed in a different ticket too, but I'm commenting here just in case.

Reply:
Also to be removed in another ticket; as a temporary measure I've currently forced siteSpecific to always return CS content.

Reply:
`siteSpecific` does not always return CS content in this PR. Is that still the intended behaviour? As long as the website works correctly when we merge the changes in this PR, then that's fine.

Reply:
Yep sorry that is in the next ticket (https://github.com/isaaccomputerscience/isaac-react-app/blob/Hannah/physics-deletion-utils/src/app/services/siteConstants.ts) as I encountered more issues in that one.  The site still works fine in this branch.